CHANGELOG.rst

```diff
@@ -2,6 +2,16 @@
 Changelog
 =========
 
+0.12.0 (2023-10-24)
+===================
+
+* Added :meth:`~zyte_common_items.base.ProbabilityMixin.get_probability` helper
+  method in page classes (e.g. :class:`~zyte_common_items.Product`,
```

Comment:
Isn't it also added for the item classes, not just for page classes?

Reply:
Ohh right, I got mixed up by the previous terminologies that were use in the previous release. I just realized it was _"page classes **for** BusinessPlace, JobPosting, ..."_ https://github.com/zytedata/zyte-common-items/pull/66/commits/cf0abd43de8cd9ec4730f76c937d506c4cd93ec0